samza-core/src/main/java/org/apache/samza/metrics/MetricGroup.java

```diff
@@ -62,6 +62,15 @@ public T getValue() {
     });
   }
 
+  public <T> Gauge<T> newGauge(String name, final ValueFunction<T> valueFunc, Boolean overwrite) {
```

Comment:
There seems to be a mixing of terminology here vs MetricsRegistry (overwrite vs overrideExistingGauge). All we're doing is passing the arg here on to the registry; is the difference in naming intentional?

Reply:
done, made naming consistent.